setup.py

```diff
@@ -49,6 +49,7 @@
     ]},
     extras_require={
         "bcrypt": ["bcrypt"],
+        "argon2": ["argon2-cffi >= 16.0.0"],
```

Comment:
16 added the argon2.low_level module which is used to set a salt. (See comments below in the review.)